__tests__/app.test.js

```diff
@@ -29,7 +29,7 @@ describe('GET /api/topics', () => {
         });
       });
   });
-  test('sends GET to endpoint -> checks object properties and values', () => {
+  test('sends GET to topics endpoint -> checks object properties and values', () => {
```

Comment:
The detail you've gone into with your tests here is awesome. I think it would be even better if you combined all these assertions into one test, but I'll leave that decision to you.